kedro/framework/session/session.py

```diff
@@ -253,11 +253,16 @@ def _get_config_loader(self) -> AbstractConfigLoader:
         extra_params = self.store.get("extra_params")
 
         config_loader_class = settings.CONFIG_LOADER_CLASS
+        config_loader_args = settings.CONFIG_LOADER_ARGS
+        if not (
+            "base_env" in config_loader_args or "default_run_env" in config_loader_args
+        ):
+            config_loader_args.update({"base_env": "base", "default_run_env": "local"})
```

Comment:
In the original issue, I propose to use session to set the default. Technically, `_ProjectSettings` feels like the right place to set it, but I don't think there are any real difference between having the defaults in `_ProjectSettings` or `KedroSession`. I think this is fine, but basically this will always override the `_ProjectSettings` and the default there isn't relevant (either override by user provided CONFIG_LOADER_ARGS or session).

@idanov was the one pointing out that we could have the defaults in `_ProjectSettings`, so I'd love to see if you have any thought on this.

https://github.com/kedro-org/kedro/issues/2971#top
```python
# Maybe update this in `session.py`?
config_loader_args = {"base_env"="base", "default_run_env"="local"}
config_loader_args.update(settings.CONFIG_LOADER_ARGS) # Overwrite if any
return config_loader_class(
            conf_source=self._conf_source,
            env=env,
            runtime_params=extra_params,
            **config_loader_args,
        )

# Original
settings.CONFIG_LOADER_CLASS
return config_loader_class(
    conf_source=self._conf_source,
    env=env,
    runtime_params=extra_params,
    **settings.CONFIG_LOADER_ARGS,
)
```

Reply:
Yeah I remember.. I don't think we considered this situation though where you override some args but not explicitly the environments..